heudiconv/tests/test_bids.py

```diff
@@ -500,6 +528,154 @@ def create_dummy_no_shim_settings_bids_session(session_path):
 
     return session_struct, expected_result, expected_fmap_groups, expected_compatible_fmaps
 
+def create_dummy_no_shim_settings_custom_label_bids_session(session_path, label_size=4, label_seed=42):
```

Comment:
Can the function `create_dummy_no_shim_settings_bids_session` be dropped and have the different tests call this function instead, once with `'ModalityAcquisitionLabel'` and once with `'CustomAcquisitionLabel'`?

Reply:
I believe so, I'll look into that !

Reply:
Ok so i looked into it, it can be done but the code may become more messy because the IntendedFor `expected_result` and `expected_compatible_fmaps` which will depend on the parameter chosen (`ModalityAcquisitionLabel` or `CustomAcquisitionLabel`).

So the advantage of merging two into one will make the code more compact, while having two separate session keep the logic of the different use cases independent (one use case when field maps are named according to different modalities, and another one according to different sequences possibly within the same modality).

Reply:
Sounds good.
Thanks for looking into it.